governance/community_architecture.md

```diff
@@ -74,6 +77,11 @@ their contributions and their expert knowledge of the Project Software and
 Services that Council Members will provide useful guidance, both technical and
 in terms of project direction, to potentially less experienced contributors.
 
+The Council will have between 4 and 7 members.
+No more than 2 Council Members can report to one person or company
+(including Institutional Partners) through employment or contracting work
+(including the reportee, i.e. the reportee + 1 is the max).
```

Comment:
We didn't have anything like this, but the contract we signed with numfocus did, so I added it